app/src/androidTest/java/com/droidhats/campuscompass/views/CalendarFragmentTest.kt

```diff
@@ -149,7 +149,7 @@ class CalendarFragmentTest {
         //checks if there exists a button with text "okay" within the calendar selection widget an performs a click on it
```

Comment:
Change "okay" to "OKAY" to be emphasize case sensitivity. In the app the text is all caps

Reply:
Good idea! This has been fixed 😄